packages/create-cloudflare/templates/hello-world/js/wrangler.toml

```diff
@@ -96,3 +96,10 @@ compatibility_flags = ["nodejs_compat"]
 # [[vectorize]]
 # binding = "MY_INDEX"
 # index_name = "my-index"
+
```

Comment:
There's already an entry for D1 on line 27. If we want to update the comment around it that's fine, but this file is copy pasted between all workers templates, and it would be nice to do it for all of them uniformally if we want to make this change

Reply:
Oh cool. I didn't realize that this was included in the long string of comments.

Reply:
I was hiding it from you 😄